### PR TITLE
Update ActiveRelationTrait.php

### PR DESCRIPTION
**Pull request by sergkash7**

### CODE REVIEW

**Comment by bizley** [comment]
What is the reason behind this?

**Comment by sergkash7** [comment]
If I'm not mistaken, then this annotation is not needed here. In addition, it is incorrect, since the method may return null. This annotation confuses the static analyzer.

**Comment by bizley** [comment]
I don't know how it could confuse the static analyzer but yes, it would be nice to have it updated to show all possible return types. But I don't think it should be removed.

**Comment by sergkash7** [comment]
Added null

File: framework/db/ActiveRelationTrait.php [hunk 1]
```diff
@@ -17,7 +17,7 @@
  * @author Carsten Brandt <mail@cebe.cc>
  * @since 2.0
  *
- * @method ActiveRecordInterface one($db = null) See [[ActiveQueryInterface::one()]] for more info.
+ * @method ?ActiveRecordInterface one($db = null) See [[ActiveQueryInterface::one()]] for more info.
```
Comment (bizley) [inline framework/db/ActiveRelationTrait.php:20]:
```suggestion
 * @method ActiveRecordInterface|array|null one($db = null) See [[ActiveQueryInterface::one()]] for more info.
```

**Comment by bizley** [comment]
Thank you.

**Pull request merged**